File: CS/GenerateUserFriendlyId.Module/BusinessObjects/IUserFriendlyIdDomainComponent.cs
using DevExpress.ExpressApp.DC;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.Validation;
using GenerateUserFriendlyId.Module;

namespace GenerateUserFriendlyId.Module.BusinessObjects {
     [DomainComponent]
    public interface IUserFriendlyIdDomainComponent : ISupportSequentialNumber { }
}

File: VB/GenerateUserFriendlyId.Module/BusinessObjects/IUserFriendlyIdDomainComponent.vb
Imports DevExpress.ExpressApp.DC
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.Validation
Imports GenerateUserFriendlyId.Module

Namespace GenerateUserFriendlyId.Module.BusinessObjects
	 <DomainComponent>
	 Public Interface IUserFriendlyIdDomainComponent
		 Inherits ISupportSequentialNumber

	 End Interface
End Namespace